tests/MongoDB.Bson.Tests/ObjectModel/Decimal128Tests.cs

```diff
@@ -61,6 +63,7 @@ public void Decimal(string valueString, string s)
 
         [Theory]
         [InlineData("0", "0")]
+        [InlineData("0.00", "0.00")]
```

Comment:
Is this the only test that failed prior to the code change?

Reply:
No, after the nearest commit (which fixed some minor bug in tests).
See also:
`[InlineData("0.000", "0.000")]`
from the `Decimal` test.

I also clarified the test result for the ticket 2001:
` [InlineData("10000.0000000000000000000000001", "10000.000000000000000000000000")] `